templates/base.html

```diff
@@ -5,20 +5,18 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <!-- Begin Jekyll SEO tag v2.8.0 -->
-        <title>{{config.title}} | {{config.description}}</title>
+        <title>{% if page.title %}{{page.title}} - {{config.title}}{% else %}| {{config.description}}{% endif %}</title>
         <meta name="generator" content="Zola v0.17.2">
-        <meta property="og:title" content="{{config.title}}">
+        <meta property="og:title" content="{% if page.title %}{{page.title}}{% else %}{{config.title}}{% endif %}">
         <meta property="og:locale" content="en_US">
-        <meta name="description" content="{{config.description}}">
-        <meta property="og:description" content="{{config.description}}">
-        <link rel="canonical" href="/">
-        <meta property="og:url" content="/">
-        <meta property="og:site_name" content="">
+        <meta name="description" content="{% if page.title %}{{page.title}}{% else %}{{config.description}}{% endif %}">
+        <meta property="og:description" content="{% if page.title %}{{page.title}}{% else %}{{config.description}}{% endif %}">
+        <meta property="og:site_name" content="{{config.title}}">
         <meta property="og:type" content="website">
         <meta name="twitter:card" content="summary">
-        <meta property="twitter:title" content="{{config.title}}">
+        <meta property="twitter:title" content="{% if page.title %}{{page.title}}{% else %}{{config.title}}{% endif %}">
         <script type="application/ld+json">
-            {"@context":"https://schema.org","@type":"WebSite","description":"{{config.description}}","headline":"{{config.title}}","name":"{{config.title}}","url":"/"}</script>
+            {"@context":"https://schema.org","@type":"WebSite","headline":"{% if page.title %}{{page.title}}{% else %}{{config.title}}{% endif %}","name":"{% if page.title %}{{page.title}}{% else %}{{config.title}}{% endif %}"}</script>
```

Comment:
I'm surprised that both the headline and name should be the same thing here.

Broadly, I suspect that this isn't the correct application of this (there's also a WebPage schema, which seems more relevant). I *think* this is intended to cover the entire site?

Reply:
You might be right, I didn't dig into it deeply. The post is getting bad previews on Mastodon, so I'd like to fix it quickly - perhaps we just take this linking data out for now until we can put it in right?

Reply:
That would be my preference